datahub/dnb_api/tasks.py

```diff
@@ -48,3 +49,34 @@ def sync_company_with_dnb(self, company_id, fields_to_update=None):
     will be synced.
     """
     _sync_company_with_dnb(company_id, fields_to_update, self)
+
+
+@shared_task(
+    acks_late=True,
+    priority=9,
+)
+def update_company(company_data, fields_to_update=None):
```

Comment:
I assume `company_data` is D&B company data – maybe `dnb_company_data` will be more self explanatory?

```suggestion
def update_company(dnb_company_data, fields_to_update=None):
```